FreeRTOS/FreeRTOS-Plus-TCP
# [IPv6] Fix checksum calculation of IPv6 with extension headers.

[IPv6] Fix checksum calculation of IPv6 with extension headers- RFC 2460 section 4.1

Description
-----------

With this change, FreeRTOS-Plus-TCP supports to calculate checksum for IPv6 packets with IPv6 extension headers.

Test Steps
-----------

Pass IPv6 TCP/UDP/ICMP checksum test cases with this change.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
